src/Nox.Types.EntityFramework/Types/DatabaseNumber/DatabaseNumberConverter.cs

```diff
@@ -2,7 +2,7 @@
 
 namespace Nox.Types.EntityFramework.Types;
 
-public class DatabaseNumberConverter : ValueConverter<DatabaseNumber, long>
+public class DatabaseNumberConverter : ValueConverter<DatabaseNumber, ulong>
```

Comment:
According to the latest changes, the underlying type is long.

Reply:
updated